tests/nn/pipe_process/test_pipe.py

```diff
@@ -25,6 +25,8 @@
 import pytest
 import torch
 from torch import nn
+from torch.optim.optimizer import Optimizer
```

Comment:
does it make sense to have the new tests in a new file?

Reply:
Yeah, good point. I will move new tests under experimental.

Reply:
@min-xu-ai : Do you suggest adding these 2 tests in a new file in the same location? Something like tests/nn/pipe_process/test_ampnet_pipe.py

Or do you think we should create a new directory tests under experimental? Something like experimental/tests/nn/pipe_process/test_ampnet_pipe.py

Reply:
As of the recent-most commit, I followed the second option. And we can run those tests using `pytest experimental/tests`

But happy to do it the other way.

Reply:
sounds good to me. thanks a lot!